src/neo/SmartContract/Native/ContractManagement.cs

```diff
@@ -120,11 +121,30 @@ public IEnumerable<ContractState> ListContracts(StoreView snapshot)
             return snapshot.Storages.Find(listContractsPrefix).Select(kvp => kvp.Value.GetInteroperable<ContractState>());
         }
 
+        [ContractMethod(0, CallFlags.WriteStates | CallFlags.AllowNotify)]
+        private ContractState CreateChild(ApplicationEngine engine, StackItem data)
```

Comment:
We don't need to create this method, right? We can create a new child contract in contract/script directly, by this following way.

```
var contract = ContractManagement.GetContract(hash)
ContractManagement.Deploy(contract.nef, contract.manifest, data)
```

Reply:
The problem is that the `sender` of the transaction could be the same.

Reply:
https://github.com/neo-project/neo/pull/2240/files#diff-c827c27f2bec36e465fcb3b3af10590d6f952a3888adedc9a7870860ebe72079R146 may fix it.